prometheus/value.go

```diff
@@ -160,3 +160,15 @@ func makeLabelPairs(desc *Desc, labelValues []string) []*dto.LabelPair {
 	sort.Sort(labelPairSorter(labelPairs))
 	return labelPairs
 }
+
+func copyLabelPairs(source []*dto.LabelPair) []*dto.LabelPair {
+	labelPairs := make([]*dto.LabelPair, 0, len(source))
+	for _, pair := range source {
+		labelPairs = append(labelPairs, &dto.LabelPair{
+			Name:  pair.Name,
+			Value: pair.Value,
+		})
```

Comment:
Do you need to append here since you created the slice with the correct length already? Why not just assign the index of the `labelPairs` to the current index in the loop?

Reply:
This was mostly a copy-paste job from the method above, as I was aiming for consistency. I don't think there is any significant difference in the two approaches, but I don't feel strongly in either direction.